### PR TITLE
FIx nameing + one var acess without lock

### PR DESCRIPTION
**Pull request by gburanov**

### CODE REVIEW

File: offset_manager.go [hunk 1]
```diff
@@ -366,7 +365,6 @@ func (pom *partitionOffsetManager) updateCommitted(offset int64, metadata string
 
 	if pom.offset == offset && pom.metadata == metadata {
 		pom.dirty = false
-		pom.offsetChangedSinceLastForced = true
```
Comment (gburanov) [inline offset_manager.go:369]: Removed this to simplify code. I think you are right @itszootime  code is simplier, and some extra commits once per hour would not harm

File: offset_manager.go [hunk 2]
```diff
@@ -450,17 +448,14 @@ func (om *offsetManager) newBrokerOffsetManager(broker *Broker) *brokerOffsetMan
 func (bom *brokerOffsetManager) mainLoop() {
 	for {
 		select {
-		case <-bom.commitForcedTimer.C:
-			if len(bom.subscriptions) > 0 {
+		case <-bom.forcedTimer.C:
+			if len(bom.subscriptions) > 0 && bom.autoCommit {
```
Comment (gburanov) [inline offset_manager.go:452]: Added bom.autoCommit check  + test

**Comment by gburanov** [comment]
@itszootime  please check, I made the necessary fixes

**Review by itszootime — APPROVED** [review]
Thanks for making these changes!

**Pull request merged**